src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -45,7 +46,7 @@ namespace AppInstaller::CLI::Execution
 
     OutputStream Reporter::GetOutputStream(Level level)
     {
-        OutputStream result = GetBasicOutputStream();
+        OutputStream result = m_standardOutputStream;

```

Comment:
By making all of these streams a copy of the original, you are not respecting changes to `m_channel` and VT enablement state.  You also are not gaining what I suspect you might think you are gaining; that closing `m_standardOutputStream` also closes all of the active streams.  Because they are all copies that will not happen.

It would be possible to create a single base object that is in between all attempts to write to the stream that the reporter is given, but my suggestion was to try to do that by operating inside the STL infrastructure.  Because you only care about being able to control whether output happens or not, it shouldn't be that hard.  Alternately, wrapping it would be fine too, but that should be done in a functional way that integrates with the existing code.